.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
```

Comment:
Curios, did you mean to name the template file `config.yml` ? or good to change to something like `etcd-questions.yml` ?

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Reply:
Thanks!